domains/khacviet.json

```
@@ -0,0 +1,10 @@
+{
+  "description": "Khac Viet Portfolio website",
+  "owner": {
+    "username": "Khac-Viet",
+    "email": "viet2ca11@gmail.com"
+  },
+  "record": {
+    "A": "103.97.126.29"
```

A record(s) should be in an array
```suggestion
    "A": ["103.97.126.29"]
```